lib/utils/date.dart

```diff
@@ -40,15 +40,11 @@ class BreezDateUtils {
   }
 
   static DateTime? blockDiffToDate({required int blockHeight, required int? expiryBlock}) {
-    if (expiryBlock != null) {
-      int diff = expiryBlock - blockHeight;
-      if (diff > 0) {
-        diff *= 600;
-        final time = DateTime.now();
-        return time.add(Duration(seconds: diff));
-      }
-    }
-    return null;
+    if (expiryBlock == null || expiryBlock <= blockHeight) return null;
+    int diff = expiryBlock - blockHeight;
+    diff *= 600;
+    final time = DateTime.now();
+    return time.add(Duration(seconds: diff));
```

Comment:
Avoid redundant calculation for `diff` variable.
```suggestion
    int diffInSecs = (expiryBlock - blockHeight) * 600;
    final time = DateTime.now();
    return time.add(Duration(seconds: diffInSecs));
```

Reply:
Added